### DISCUSSION TITLE
How to output files into exe directory?

### DISCUSSION THREAD

**Discussion by serg06** [microsoft/vcpkg#18617] [Q&A]
I'm trying to create a new port #18613 , but it requires some `.bin`, `.pak`, and `.dat` files to exist in the .exe's directory, or else the library doesn't work. Is it possible to do that with with vcpkg?

I tried putting some `.bin` files into `${CURRENT_PACKAGES_DIR}/bin`, but they don't get copied over to the `.exe` directory.

**Comment by JackBoosY** [comment]
Generally, the sources's cmake build system should contains `INSTALL(FILES)`, `INSTALL(TARGETS)` or other `INSTALL` command in `CMakeLists.txt` / `*.cmake`, that vcpkg can use `vcpkg_cmake_install` / `vcpkg_install_cmake` to install the selected files automaticly.

**Reply by serg06** [reply]
@JackBoosY Ohh. Unfortunately it doesn't contain any calls to `INSTALL(...)`.

```
$ find . -regextype posix-extended -iregex '.*\.(txt|cmake)' -exec grep -i "INSTALL(" {} \;
$
```

Is that something I should patch in?

**Reply by JackBoosY** [reply]
Yes, add `install(FILES)` and `install(TAGETS)` to the source's `CMakeLists.txt`.
